docker/docker/src/main/java/org/arquillian/cube/docker/impl/client/CubeDockerConfigurator.java

```diff
@@ -61,18 +64,22 @@ public void configure(@Observes CubeConfiguration event, ArquillianDescriptor ar
     }
 
     private void configure(ArquillianDescriptor arquillianDescriptor) {
-        operatingSystemFamilyInstanceProducer.set(new OperatingSystemResolver().currentOperatingSystem().getFamily());
-        Map<String, String> config = arquillianDescriptor.extension(EXTENSION_NAME).getExtensionProperties();
-        CubeDockerConfigurationResolver resolver = new CubeDockerConfigurationResolver(topInstance.get(),
-            dockerMachineInstance.get(),
-            boot2DockerInstance.get(),
-            operatingSystemFamilyInstanceProducer.get());
-        resolver.resolve(config);
-        CubeDockerConfiguration cubeConfiguration = CubeDockerConfiguration.fromMap(config, injectorInstance.get());
-        cubeConfiguration = resolveDynamicNames(cubeConfiguration);
-        System.out.println(cubeConfiguration);
-        hostUriContextInstanceProducer.set(new HostIpContext(cubeConfiguration.getDockerServerIp()));
-        configurationProducer.set(cubeConfiguration);
+        try {
+            operatingSystemFamilyInstanceProducer.set(new OperatingSystemResolver().currentOperatingSystem().getFamily());
+            Map<String, String> config = arquillianDescriptor.extension(EXTENSION_NAME).getExtensionProperties();
+            CubeDockerConfigurationResolver resolver = new CubeDockerConfigurationResolver(topInstance.get(),
+                dockerMachineInstance.get(),
+                boot2DockerInstance.get(),
+                operatingSystemFamilyInstanceProducer.get());
+            resolver.resolve(config);
+            CubeDockerConfiguration cubeConfiguration = CubeDockerConfiguration.fromMap(config, injectorInstance.get());
+            cubeConfiguration = resolveDynamicNames(cubeConfiguration);
+            System.out.println(cubeConfiguration);
+            hostUriContextInstanceProducer.set(new HostIpContext(cubeConfiguration.getDockerServerIp()));
+            configurationProducer.set(cubeConfiguration);
+        } catch (ExecutionException e) {
+            log.log(Level.WARNING, "Error executing docker command", e);
```

Comment:
My reason for not throwing RuntimeException here is for the scenario I outlined in #664.

For example, in my case I have a test suite where some tests that use Cube and are annotated with `@RequiresDocker`. Also other tests that do not make use of Cube.

If Docker is not available, I'd expect the Cube tests to be skipped and the non-cube tests to proceed to run as normal. This is not possible at the moment because the ExecutionException breaks _every_ test.

The alternative to this change would be for me to split the Cube tests into their own module, but I wanted to avoid doing that if possible.

Hope that clarifies things a little.